ROADMAP.md

```diff
@@ -4,32 +4,37 @@ This document defines a high level roadmap for Karmada development and upcoming
 Community and contributor involvement is vital for successfully implementing all desired items for each release.
 We hope that the items listed below will inspire further engagement from the community to keep karmada progressing and shipping exciting and valuable features.
 
+## 2022 H1
+- Multi-cluster HA scheduling policy
+  * spread by region
+  * spread by zone
+  * spread by provider
+- Multi-cluster Ingress
+- Multi-cluster HPA (Horizontal Pod Autoscaling)
+- Federated resource quota
+- API reference
+- [Karmada website](https://karmada.io/) refactor
+- Policy-based governance, risk, and compliance
+- Multi-cluster DNS (cluster identity)
+- Global search across clusters
+- Scheduling re-balancing
```

Comment:
`re-balancing` you mean adjust resource pressure over clusters?

Reply:
Yes, more or less. After the normal schedule progress, sometimes we need to make some adjustments, that's basically the [karmada-deschedule](https://github.com/karmada-io/karmada/blob/master/docs/descheduler.md).

Reply:
In my opinion, `re-balancing` contains two aspect:
1. adjust unschedulable worload, just as [karmada-deschedule](https://github.com/karmada-io/karmada/blob/master/docs/descheduler.md) do.
2.  balance the pressure of clusters. Workloads are scheduled and running, but clusterA MEM usage is high, while clusterB is in low level. Then we shall drift some workloads to other cluster.

Reply:
I think `balance the pressure of clusters.` is also the scope of `karmada-deschedule`. Am I right @Garrybest ?

Reply:
That's right. This should be a part of job for descheduler. See https://github.com/karmada-io/karmada/blob/2d47c5b08a27132b2c2e2fa6ef080f54ee157c92/docs/proposals/scheduling/697-descheduler/README.md. However, this is not implemented yet.